app/assets/javascripts/admin/addon/components/schema-theme-setting/editor/tree-node.gjs

```diff
@@ -1,11 +1,11 @@
 import Component from "@glimmer/component";
 import { tracked } from "@glimmer/tracking";
-import { fn, get } from "@ember/helper";
+import { get } from "@ember/helper";
 import { on } from "@ember/modifier";
-import { action } from "@ember/object";
-import didInsert from "@ember/render-modifiers/modifiers/did-insert";
 import { gt } from "truth-helpers";
-import dIcon from "discourse/helpers/d-icon";
+import concatClass from "discourse/helpers/concat-class";
+import icon from "discourse/helpers/d-icon";
```

Comment:
Is this recent guidance? I have still been using `dIcon`

Reply:
I'd say so: it's more readable AND you save 1 byte each time! 😝

Reply:
Bu-bu-it's `d-icon` not `icon` :sob: Will do a quick pass through admin to get rid of it.